src/test/java/edition/academy/seventh/serivce/BookMapperTest.java

```diff
@@ -14,7 +14,7 @@
  * @author Bartosz Kupajski
  * @author Kamil Rojek
  */
-public class ITBookMapperTest {
+public class BookMapperTest {
```

Comment:
Nie wiem czy jest to zasadna zmiana nazwy. O ile zmiana "IT" na "It" była by, to ten test ma sprawdzać konkretną implementację dla konkretnej księgarnii jeżeli dobrze rozumiem.

Reply:
ale konkretne nazwa księgarni pokrywa nam się z nazwą sprawdzaną przez nową wtyczkę, która za jej pomocą oznacza Testy Integracyjne